weblate_web/templates/donate.html

```diff
@@ -2,7 +2,7 @@
 {% load static %}
 {% load i18n %}
 {% block title %}{% trans "Donate to Weblate" %}{% endblock %}
-{% block description %}{% blocktrans %}Weblate is provided free of charge and welcomes donations. There are many ways to contribute. Discuss it with your associates.{% endblocktrans %}{% endblock %}
+{% block description %}{% blocktrans %}Weblate does not require payment, and welcomes donations. There are many ways to contribute. Discuss it with your associates.{% endblocktrans %}{% endblock %}
```

Comment:
What about "and welcomes contributions"?

Reply:
As this is on the donate page, I'd probably stick with donations here.